clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -131,9 +131,9 @@ abstract class OpenstackClientProvider {
    * @param timeoutMins
    * @return
    */
-  void deploy(String stackName, String heatTemplate, Map<String, String> parameters, boolean disableRollback, Long timeoutMins) {
+  void deploy(String region, String stackName, String heatTemplate, Map<String, String> parameters, boolean disableRollback, Long timeoutMins) {
     try {
-      client.heat().stacks().create(stackName, heatTemplate, parameters, disableRollback, timeoutMins)
+      client.useRegion(region).heat().stacks().create(stackName, heatTemplate, parameters, disableRollback, timeoutMins)
```

Comment:
I know Derick added  a `getRegionClient(region)` method which can be used in place of `client.useRegion(region), but it might not have been merged yet.